independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapMavenContext.java

```diff
@@ -616,7 +617,8 @@ private Model loadCurrentProjectModel() throws BootstrapMavenException {
                 .addProfileActivator(new PropertyProfileActivator())
                 .addProfileActivator(new JdkVersionProfileActivator())
                 .addProfileActivator(new OperatingSystemProfileActivator())
-                .addProfileActivator(new FileProfileActivator().setPathTranslator(new DefaultPathTranslator()));
+                .addProfileActivator(new FileProfileActivator().setProfileActivationFilePathInterpolator(
+                        new ProfileActivationFilePathInterpolator().setPathTranslator(new DefaultPathTranslator())));
```

Comment:
See https://github.com/apache/maven/pull/649

Reply:
@famod do you know whether this change is backward compatible? Would this work in Maven 3.6.2. for example?